gradle/config/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+    "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<!-- Empty suppressions file: copy it into gradle/config/checkstyle and specify required suppressions
+     (https://checkstyle.sourceforge.io/config_filters.html#SuppressionFilter)-->
+<suppressions>
+
```

Comment:
I had to add `<suppress checks="[a-zA-Z0-9]*" files="[\\/]generated[\\/]"/>` to skip all  generated files.